packages/cli/src/wizard/wizard.js

```diff
@@ -7,13 +7,21 @@
 
 const inquirer = require('inquirer');
 const ApiClient = require('@lhci/utils/src/api-client.js');
+const {
+  loadRcFile,
+  flattenRcToConfig,
+  resolveRcFilePath,
+} = require('@lhci/utils/src/lighthouserc.js');
+const _ = require('@lhci/utils/src/lodash.js');
 const log = require('lighthouse-logger');
 
 /**
  * @param {import('yargs').Argv} yargs
  */
 function buildCommand(yargs) {
-  return yargs;
+  return yargs.options({
+    config: {description: 'The lighthouserc.json file preferences.'},
```

Comment:
this option should already be set in `src/cli.js` for all commands, so shouldn't be necessary here.

Reply:
Thank yes its not needed i change that back to return yargs;